client-tools/Shell/ProcessMonitoringFeature.cpp

```diff
@@ -96,8 +96,12 @@ ProcessMonitoringFeature::~ProcessMonitoringFeature() = default;
 
 void ProcessMonitoringFeature::validateOptions(
     std::shared_ptr<options::ProgramOptions> /*options*/) {
+#ifdef ARANGODB_ENABLE_MAINTAINER_MODE
   _enabled =
       server().getFeature<V8SecurityFeature>().isAllowedToControlProcesses();
+#else
+  _enabled =
```

Comment:
I except such ifdef for whole Feature in `client-tools/Shell/arangosh.h`.
In cmake also should be if to don't build this files in such case

Reply:
this would break the js API.